tests/activity-report.spec.ts

```diff
@@ -397,4 +397,16 @@ test.describe("Activity Report", () => {
     // verify the link works by checking whether the recipients are visible
     await expect(page.getByText(`${recipients}`)).toBeVisible();
   });
+
+  test('switching objectives properly clears all fields', async ({ page}) => {
+    await page.goto('http://localhost:3000/');
+    // create a new report
+    await page.getByRole('link', { name: 'Activity Reports' }).click();
+    await page.getByRole('button', { name: '+ New Activity Report' }).click();
+
+    // select a recipient
+    await page.getByRole('group', { name: 'Was this activity for a recipient or other entity? *' }).locator('label').filter({ hasText: 'Recipient' }).click();
+    await page.locator('#activityRecipients div').filter({ hasText: '- Select -' }).nth(1).click();
+    await page.locator('#react-select-3-option-0-0').click();
+  });
 });
```

Comment:
This is not part of this PR,  but got included as a result of fixing an incorrect merge.